public/index.html

```diff
@@ -3,7 +3,7 @@
   <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
-    <meta name="theme-color" content="#000000" />
+    <meta property="csp-nonce" content="{{ styleNonce }}" />
```

Comment:
inject a nonce for certifying style elements generated by a material-ui dependency (JSS)